### PR TITLE
johnny robohand bundle balance pass

### PR DESCRIPTION
**Pull request by dawsonkeyes**
## About The Pull Request

This PR rebalances the Robohand bundle while also fixing faulty spritework for the clothing and entirely respriting the Malorian Arms 3516 pistol altogether.

image

## How This Contributes To The Skyrat Roleplay Experience

The kit itself is an interesting change of pace compared to the otherwise versatile nature of other available bundles. It no longer offers excess weaponry in the form of an energy sword arm, nor do the bullets instantly embed/dismember opponents, but the pistol remains a strong weapon to rely on all the same. This is not readied to the uplink, because we’d need to see another TC volume change before that happens, but it is under OPFOR otherwise.

## Changelog

:cl:
balance: rebalanced the robohand kit for normal gameplay
/:cl:

### CODE REVIEW

**Comment by dawsonkeyes** [comment]
a more comprehensive list of changes
- 3516 14mm rounds no longer have any changed embed or delimb chance and, save for their wall/window penetration, are the exact same as .357 rounds now
- the bundle itself has two more magazines for a total of six, fully filling the provided belt
- the pants and vest now have their proper sprites
- the pants, vest, and glasses have been given wound armor values of 100, but are otherwise unchanged from the originals armor-wise
- esword arm has been entirely gutted from the kit, get fucked
- the robot arm is now the _left arm_ instead of the right
- the malorian 3516 has a unique manufacturer, as it should

**Comment by dawsonkeyes** [comment]
for some reason the malorian was scarborough, so i made it, you know, malorian
image

File: modular_skyrat/modules/robohand/code/robohand.dm [hunk 1]
```diff
@@ -108,10 +113,10 @@
 	caliber = CALIBRE_14MM
 	projectile_type = /obj/projectile/bullet/c14mm
 
-/obj/projectile/bullet/c14mm
+/obj/projectile/bullet/c14mm //vanilla .357 but pierces everything except mobs. groovy
 	name = "14mm bullet"
 	damage = 60
```
Comment (tf-4) [inline modular_skyrat/modules/robohand/code/robohand.dm]: Drop the damage a little

File: modular_skyrat/modules/robohand/code/silverhand_bundle.dm [hunk 2]
```diff
@@ -4,37 +4,47 @@
...
-	name = "badass pants"
-	desc = "Strangely firm yet soft black pants, these appear to have some armor padding for added protection."
-	armor = list(MELEE = 20, BULLET = 20, LASER = 20, ENERGY = 20, BOMB = 20, BIO = 0, FIRE = 0, ACID = 0, WOUND = 5)
+/obj/item/clothing/under/pants/track/robohand
+	name = "rockerboy pants"
+	desc = "Extremely comfortable pants custom-made in the memory of Johnny Robohand, padded with kevlar for your enjoyment."
+	special_desc = "A unique cyberlink allows these pants to somehow protect most of your body, and a specialized weave makes them resistant to wounding attacks."
+	special_desc_requirement = EXAMINE_CHECK_SYNDICATE
+	special_desc_affiliation = "memories of Johnny Robohand"
+	armor = list(MELEE = 20, BULLET = 20, LASER = 20, ENERGY = 20, BOMB = 20, BIO = 0, FIRE = 0, ACID = 0, WOUND = 100)
```
Comment (tf-4) [inline modular_skyrat/modules/robohand/code/silverhand_bundle.dm]: None of this needs 100 wound armour

File: modular_skyrat/modules/robohand/code/silverhand_bundle.dm [hunk 3]
```diff
@@ -4,37 +4,47 @@
...
-	name = "badass sunglasses"
-	desc = "Strangely ancient technology used to help provide rudimentary eye cover. Enhanced shielding blocks flashes. These ones seem to be bulletproof?"
+	name = "badass shades"
+	desc = "An awesome pair of aviator shades that provide eye cover. Enhanced shielding blocks flashes, and a reinforced frame somehow makes these bullet-resistant!"
+	special_desc = "Provides the exact same protection as a bulletproof helmet, and resists wounds for the wearer. You'd be a menace if you could find a helmet to wear WITH these, but you'd look less cool."
+	special_desc_requirement = EXAMINE_CHECK_SYNDICATE
+	special_desc_affiliation = "memories of Johnny Robohand"
 	body_parts_covered = HEAD
-	armor = list(MELEE = 20, BULLET = 60, LASER = 20, ENERGY = 20, BOMB = 20, BIO = 0, FIRE = 0, ACID = 0, WOUND = 5)
+	armor = list(MELEE = 20, BULLET = 60, LASER = 20, ENERGY = 20, BOMB = 20, BIO = 0, FIRE = 0, ACID = 0, WOUND = 100)
```
Comment (tf-4) [inline modular_skyrat/modules/robohand/code/silverhand_bundle.dm]: Lower the protections a _lot_ on this

File: modular_skyrat/modules/robohand/code/silverhand_bundle.dm [hunk 4]
```diff
@@ -4,37 +4,47 @@
...
 //Again, not a bug, it's a feature. ALL PARTS COVERED!!
-/obj/item/clothing/suit/armor/bulletproof/robohand
-	name = "badass bulletproof armor"
+/obj/item/clothing/suit/armor/bulletproof/old/robohand
+	name = "custom-fit ballistic vest"
 	desc = "One of a kind bulletproof armor custom made to fit one person, Johnny Robohand. It seems that your arms are protected by it as well!"
+	special_desc = "A damn badass piece of equipment that also allows the wearer to resist wounds from gunfire, melee assault, and more."
+	special_desc_requirement = EXAMINE_CHECK_SYNDICATE
+	special_desc_affiliation = "memories of Johnny Robohand"
+	armor = list(MELEE = 15, BULLET = 60, LASER = 10, ENERGY = 10, BOMB = 40, BIO = 0, FIRE = 50, ACID = 50, WOUND = 100) //exact same as the normal bulletproof vest but wound-resistant
```
Comment (tf-4) [inline modular_skyrat/modules/robohand/code/silverhand_bundle.dm]: Still doesn't need boosted wound armour

**Review by tf-4 — CHANGES_REQUESTED** [review]
I'd rather just see it removed, but here we are.

**Comment by Gandalf2k15** [comment]
That new sprite really ain't it chief

**Comment by dawsonkeyes** [comment]
> That new sprite really ain't it chief

it's not good, but it closer resembles the original gun than the first sprite /shrug

**Comment by Zergspower** [comment]
I dont feel this needs to be in the uplink at all, given the revolver costs 12 alone and this comes with a LOT of extra stuff.

**Comment by Zergspower** [comment]
77 rounds of a slightly weaker 357 plus all additional gear just doesn't belong in here for 20tc, 
For reference, left is 20tc and right is 19

image

I'm also confused by the arm swap? as from what i understand the base arm doesn't provide the same as our mutant robolimbs which is needed on the left side for charging

**Comment by Zonespace27** [comment]
> I'm also confused by the arm swap? as from what i understand the base arm doesn't provide the same as our mutant robolimbs which is needed on the left side for charging

The arm change i can understand because Johnny Silverhand's robotic arm is their left, not the right

**Comment by dawsonkeyes** [comment]
> 77 rounds of a slightly weaker 357 plus all additional gear just doesn't belong in here for 20tc, For reference, left is 20tc and right is 19
> 
> image
> 
> I'm also confused by the arm swap? as from what i understand the base arm doesn't provide the same as our mutant robolimbs which is needed on the left side for charging

General consensus seems to be that this belongs in opfor if it remains ingame at all, which sounds about right.

**Pull request closed without merging**